astropy/modeling/utils.py

```diff
@@ -14,7 +14,7 @@
 from ..extern import six
 from ..extern.six.moves import range, zip_longest, zip
 
-from ..utils import isiterable
+from ..utils import isiterable, check_broadcast
```

Comment:
I am not always sure about drawing the line in what is "public", but there could be code like `from astropy.modeling.utils import IncompatibleShapeError` out there, especially for something that uses `check_broadcast` and then wants to catch the error.  Agreed that none of this is actually in the public API so I don't feel too strongly.